Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM rust:1.73-alpine as builder
+
+WORKDIR /app
+
+# Install the latest version of the Rust toolchain.
+RUN apk update && \
+    apk upgrade
+
+# Install the musl target so we can build static binaries.
+RUN apk add --no-cache musl-dev
+
+COPY Cargo.* ./
+COPY src ./src
+
+# Build the application.
+RUN cargo build --release
```

Comment:
This is probably ok, but just FYI the `--release` flag creates an optimized build and therefore leads to much longer compile times.
Ideally, for development we would just want to run `cargo build` without the release flag, but it doesn't matter too much.

Reply:
Please note that there are 2 Dokerfiles as mantioned above, one for development which doest care for the release flag

Reply:
Are you sure?
I only see 4 files in the changes:
 - .dockerignore
 - Dockerfile
 - README.md
 - docker-entrypoint.sh

Reply:
Damn, I forgot to add it -_-
My bad. pushed.